doc/deployment-process.md

```diff
@@ -0,0 +1,57 @@
+# Deployment process
+
+## Production
+
+As outlined in the [dxw development workflow guide](http://playbook.dxw.com/#/guides/development-workflow?id=deploying), production deploys are done by manually merging develop into master. To give us a slightly more formal process around what gets deployed and when and also to give us visibility into the things that have been deployed, we additionally follow these steps when releasing to production:
+
+Releases are documented in the [CHANGELOG](CHANGELOG.md) following the [Keep a changelog](https://keepachangelog.com/en/1.0.0/) format.
+
+When a new release is deployed to production, a new second-level heading should be created in CHANGELOG.md with the release number and details of what has changed in this release.
+
+The heading should link to a Github URL at the bottom of the file, which shows the differences between the current release and the previous one. For example:
+
+### Example
+
+```
+## [release-1]
+- A change
+- Another change
+
+[release-1]: https://github.com/UKGovernmentBEIS/beis-report-official-development-assistance/compare/release-0...release-1
+```
+
+### Steps
+
+1. Create a release branch and make a pull request
+  * Create a branch from develop for the release called release-X where X is the release number
+  * Update CHANGELOG.md to:
+    document the changes in this release in a bullet point form
+    add a link to the diff at the bottom of the file
+  * Document the changes in the commit message as well
+  * git push <your-branch-name>, e.g. git push release-45
+  * Create a tag for the release in the format release-X
```

Comment:
When does the tag get pushed?

Reply:
Good spot.

I will swap them round and add `--follow-tags` to the push command!